CIPs/CIP-85/CIP-85.md

```diff
@@ -17,46 +17,50 @@ Provide schemas and associated tools to help interact with large lists in Cerami
 
 ## Abstract
 
-This CIP presents a way to store a possibly large list of items split into multiple documents based on two complementary schemas: `AppendCollection` and `CollectionSlice`.
-This allows to create a virtually infinite list of items such as a feed by leveraging multiple Ceramic documents rather than a single one.
+This CIP presents a way to store a possibly large list of items split into multiple streams based on two complementary schemas: `AppendCollection` and `CollectionSlice`.
+This allows to create a virtually infinite list of items such as a feed by leveraging multiple Ceramic streams rather than a single one.
 
 ## Motivation
 
-Storing a list of items in a single Ceramic document can make the document grow large in size, possibly exceeding Ceramic's internal limits.
+Storing a list of items in a single Ceramic stream can make the stream grow large in size, possibly exceeding Ceramic's internal limits.
```

Comment:
This is true for Tiles today, but it not really true for future stream implementations. I would add wording about that this is a specific problem for Tiles.